DFC.ServiceTaxonomy.ApiFunction/DFC.ServiceTaxonomy.ApiFunction.csproj

```diff
@@ -1,11 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
According to the [docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions),  v2 of Azure Functions only supports netcoreapp2.2.
We should either rollback to netcoreapp2.2, or upgrade to v3 of Azure Functions (which is currently in preview).